schema-registry/src/main/scala/net.manub.embeddedkafka/schemaregistry/streams/EmbeddedKafkaStreamsWithSchemaRegistryAllInOne.scala

```diff
@@ -0,0 +1,37 @@
+package net.manub.embeddedkafka.schemaregistry.streams
+
+import net.manub.embeddedkafka.Consumers
+import net.manub.embeddedkafka.schemaregistry.EmbeddedKafkaConfigWithSchemaRegistry
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.apache.kafka.streams.Topology
+import org.scalatest.Suite
+
+/** Convenience trait for testing Kafka Streams with ScalaTest.
+  * It exposes `EmbeddedKafkaStreams.runStreams` as well as `Consumers` api
+  * for easily creating and querying consumers in tests.
+  *
+  * @see [[Consumers]]
+  * @see [[EmbeddedKafkaStreamsWithSchemaRegistry]]
+  */
+trait EmbeddedKafkaStreamsWithSchemaRegistryAllInOne
```

Comment:
Could we drop the "WithSchemaRegistry" part since this class is already on a different package?